### DISCUSSION TITLE
Elements!

### DISCUSSION THREAD

**Discussion by CkreaM** [TodePond/CellPond#320] [Show and tell]
I made a cool element project, select elements from the first rule and place them down! This is v1.4.
1.2 Update patch notes; Added Cold water. Organization changes.
1.3 Update patch notes; Added Tsunami cell. Organization changes.
1.4 Update patch notes; Fixed bug where grass would be able to grow on cold water.
 =======================================================================================
# [Predator Prey Simulation Cellpond.json](https://github.com/TodePond/CellPond/files/15047958/Predator.Prey.Simulation.Cellpond.json)

**Comment by protoknight12** [comment]
Nice
